### DISCUSSION TITLE
How to download and import a macro

### DISCUSSION THREAD

**Discussion by Slammpreal** [maxnut/GDmenu#216] [Q&A]
I know you can record macros, but if I download a macro file how do I import it into GDMegaOverlay?

**Comment by ojiie** [comment]
idk u can ask maxnut to create a channel for macro, or i can send u one

**Comment by Ojiie55** [comment]
u can just click on the load button on the macro part, and open the macro folder

**Reply by Slammpreal** [reply]
Tried that does not show up in the menu when i put the macro in

**Comment by Stronkkey** [comment]
What format is the macro file (extension)?

**Reply by Slammpreal** [reply]
mhr, i think its gdr but i cant find any macros that use that type or a converter for it

**Reply by Stronkkey** [reply]
Move it into the "macros" folder inside "GDMO" which is inside your geometry dash folder.

**Comment by Ojiie55** [comment]
its .gdr

Le mer. 14 févr. 2024 à 12:49, Stronkkey ***@***.***> a
écrit :
